src/scrabble.js

```diff
@@ -1,5 +1,191 @@
-function scrabble() {
+function normalScore(aWord) {
+  if (aWord === '' || aWord === ' \t\n' || aWord === null) {
+    return 0
+  } else {
+    const newWord = aWord
+    console.log('new word is', newWord)
+    let sum = 0
+    for (i = 0; i < newWord.length; i++) {
+      if (
+        newWord[i].includes('A') ||
+        newWord[i].includes('E') ||
+        newWord[i].includes('I') ||
+        newWord[i].includes('O') ||
+        newWord[i].includes('U') ||
+        newWord[i].includes('L') ||
+        newWord[i].includes('N') ||
+        newWord[i].includes('R') ||
+        newWord[i].includes('S') ||
+        newWord[i].includes('T') ||
+        newWord[i].includes('a') ||
+        newWord[i].includes('e') ||
```

Comment:
I think you can avoid rewriting the letters again in lowercase by writing on line 5:
const newWord = aWord.toUpperCase()

Then any string that is passed in the argument  regardless of whether it has 1 upper case or all lowercase will all be returned uppercase.

Reply:
What Marc said! Instead of having to declare values for the uppercase and lowercase letters you could just transform your entered word in to upper or lower case and assigned it to a new constant.